docs/service-brokers/docs/003-overview-gcp-broker.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Google Cloud Platform Broker
+type: Overview
+---
+
+>**NOTE:** The Google Cloud Platform (GCP) Service Broker is in the experimental phase.
+
+Google Cloud Platform (GCP) Service Broker is an implementation of the [Open Service Broker](https://www.openservicebrokerapi.org/) (OSB) API hosted on GCP. It simplifies the delivery of GCP services to applications that run on Kyma. By creating GCP resources and managing their corresponding permissions, Service Broker makes it easy to consume GCP services from within a Kubernetes cluster.
+
+Kyma provides Namespace-scoped GCP Service Broker. In each Namespace, you can configure the GCP Broker against different Google Cloud Platforms. Install the GCP Service Broker by provisioning the **Google Cloud Platform Service Broker Provider** class provided by the Helm Broker. 
+
+Configure the Helm Broker to activate the **Google Cloud Platform Service Broker Provider** class:
```

Comment:
from: "Configure the Helm Broker to activate",
to: "Configure the Helm Broker to enable"